try {
    console.log(1)
} catch (error) {
    console.log(2)
} finally {
    console.log(3)
}